BEP7.md

```diff
@@ -0,0 +1,320 @@
+- Add the tokens to the destination addresses
+
+#### 6.2.3 Freeze Tokens
+
+A Binance Chain user could freeze some amount of tokens in his own address. The freeze transaction will lock his fund, thus this portion of tokens could not be used for the transactions, such as: creating orders, transferring to another account, paying fees and etc.  
+
+**Data Structure** **for Freeze Operation**: A data structure is needed to represent the freeze operation
+
+| **Field** | **Type** | **Description**                                                 |
+| :--------- | :-------- | :------------------------------------------------------------ |
+| Denom     | string   | The symbol of a collection - e.g. NNB-B90                       |
+| ID        | string   | The id of a token in the collection                             |
+
+**Freeze Process:**
+
+- Address-holder signed a freeze transaction and make it broadcasted to one of Binance Chain nodes
+- The Binance Chain node will check this transaction. If there is no error, then this transaction will be broadcasted to other Binance Chain nodes
+- Freeze transaction is committed on the blockchain by block proposer
+- Validators will verify the transaction initiator’s balance is no less than the frozen amount. The fee will be deducted from the transaction initiator’s address.  
+- This amount of tokens in the address of the transaction initiator will be moved from balance to frozen.
+
+#### 6.2.4 Unfreeze Tokens
+
+Unfreezing is to unlock some of the frozen tokens in the user's account and make them liquid again.
+
+**Data Structure** **for Unfreeze Operation**: A data structure is needed to represent the freeze/unfreeze operation
+
+| **Field** | **Type** | **Description**                                              |
+| :--------- | :-------- | :------------------------------------------------------------ |
+| Denom     | string   | The symbol of a collection - e.g. NNB-B90                       |
+| ID        | string   | The id of a token in the collection                             |
+
+**Unfreeze Process:**
+
+- Address-holder signed an unfreeze transaction and make it broadcasted to one of Binance Chain nodes
+- The Binance Chain node will check this transaction. If there is no error, then this transaction will be broadcasted to other Binance Chain nodes
+- Unfreeze transaction is committed on the blockchain by block proposer
+- Validators will verify the transaction initiator’s frozen balance is no less than the required amount. The fee will be deducted from the address of the transaction source.
+- This amount of token will be moved from frozen to balance in the transaction initiator’s address.
+
+## 7. Token Use Case Examples
+
+With Non-fungible Tokens, we can expand the token use cases on Binance Chain to be:
+
+- Event Ticket
+- VIP Privilege Status
+- Collectibles Assets
+- more ...
```

Comment:
"Virtual ATM In-Game Video Game Dex/Asset exchange"

Reply:
yea, but here is talking about `token` use case, not really about what applications, for example, 
Virtual ATM, In-Game Video Game or Dex/Asset exchange all counted under collectible assets

Reply:
Yeah, the call tonight is talking with @okwme about Application Specifics in regards to the BEP7